tesseract_common/test/plugin_loader_unit.cpp

```diff
@@ -80,7 +80,7 @@ TEST(TesseractClassLoaderUnit, LoadTestPlugin)  // NOLINT
   }
 
 // For some reason on Ubuntu 18.04 it does not search the current directory when only the library name is provided
-#if BOOST_VERSION > 106800
+#if BOOST_VERSION > 106800 && !__APPLE__
```

Comment:
Why does this not work on MacOS?

Reply:
Not sure but the other loader tests are working